roles/confluent.test/molecule/confluent-kafka-kerberos-customcerts-rhel/molecule.yml

```diff
@@ -0,0 +1,263 @@
+---
+driver:
+  name: docker
+platforms:
+  - name: kerberos1
+    hostname: kerberos1.confluent
+    groups:
+      - kerberos_server
+    image: geerlingguy/docker-centos7-ansible
+    dockerfile: ../Dockerfile.j2
+    command: ""
+    volumes:
+      - /sys/fs/cgroup:/sys/fs/cgroup:ro
+    privileged: true
+    networks:
+      - name: confluent
+  - name: zookeeper1
```

Comment:
can we test colocated zk and kafka w this one as well?

Reply:
@domenicbove I have another bug open that I plan on working on after this, regarding upgrades and uninstall packages.  This directly relates to having co-located components, and I will implement the checks in that work.  So on this PR, I think it's best to leave it out.

Reply:
There is a bigger issue here, which is that when we upgrade we leave older packages around which can cause conflicts.    This directly impacts colocated components, as if you remove packages while another service is running, you could have problems.  So it makes sense to do this as a separate issue.  I would suggest you either wait to merge your PR until that work is complete or go ahead and merge now, as it's a fairly minor change.

Reply:
It is for setting up the upgrade testing in 600-post, however the colocated components issue is new, and I had actually planned to fix the issue your PR resolves in the bug I filed for this.  So if you want, you can go ahead and merge this, as its pretty easy to check as is, or you can wait until the additional work is complete around the bug fix.